### PR TITLE
Updated Jenkins Devops

### PR DESCRIPTION
**Pull request by SoumikSen10**
## What does this PR do?
Add resource(s) 

### Description
Added a you tube course which is very good.

### Why is this valuable (or not)?
It is valuable because it gives much insights on Devops.

### How do we know it's really free?
Because it's a youtube course and hence it is free

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by soniyaprasad77** [comment]
Ensure that every check is successful.  in your case all *s has been converted into +s. do update it!

**Comment by SoumikSen10** [comment]
Can you please help me resolve this error that I am getting.

free-programming-books-lint / build (pull_request) Failing after 15s

After opening it this is what I see:
Run fpb-lint ./courses/
  fpb-lint ./courses/
  shell: /usr/bin/bash -e {0}
/home/runner/work/free-programming-books/free-programming-books/courses/free-courses-en.md
  596:1-603:1[2](https://github.com/EbookFoundation/free-programming-books/actions/runs/6487220314/job/17617035126?pr=10182#step:7:2)1  warning  Alphabetical ordering: swap l.[5](https://github.com/EbookFoundation/free-programming-books/actions/runs/6487220314/job/17617035126?pr=10182#step:7:6)99 and l.59[8](https://github.com/EbookFoundation/free-programming-books/actions/runs/6487220314/job/17617035126?pr=10182#step:7:9)  alphabetize-lists  remark-lint

⚠ 1 warning
Error: Process completed with exit code 1.

**Comment by eshellman** [comment]
I think you need to update from main

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -593,8 +593,9 @@
 
 #### Jenkins
 
-* [FREE Advanced Jenkins in K8s (Docker in Docker)](https://www.udemy.com/course/free-advanced-jenkins-in-k8s-docker-in-docker) -  CS Career Kaizen (Udemy)
+* [FREE Advanced Jenkins in K8s (Docker in Docker)](https://www.udemy.com/course/free-advanced-jenkins-in-k8s-docker-in-docker) - CS Career Kaizen (Udemy)
 * [Jenkins](https://youtube.com/playlist?list=PLhW3qG5bs-L_ZCOA4zNPSoGbnVQ-rp_dG) - Automation Step by Step
+* [Jenkins Devops](https://youtube.com/playlist?list=PL87padGs1bp9X95IxGQtFW44qWzywhbHM&si=tBgw4dwt8FYwobse) - Java Express
```
Comment (3R1Dev) [inline courses/free-courses-en.md:598]: Swap this with line 599 so that it is in alphabetical order, then you will pass the last check.

**Pull request closed without merging**